test/test-container-start-command.js

```diff
@@ -10,7 +10,7 @@ function container() {
 }
 
 tap.test('start default', function(t) {
-  var s = new container();
+  var s = newContainer();
```

Comment:
This seems odd, previous was perfectly valid js, it's OK in js for constructors to return objects, and perfectly well defined. I think this particular change is stepping over into territory I don't agree with. Thoughts? Is it a corner case of a more useful rule? Can we disable something?